ImageProcessingRequest_api

```diff
@@ -0,0 +1,86 @@
+v1.ImageProcessingRequest
+
+This is an object that describes request to the ChRIS system to be able to process an image into a desired format
+
+Fields:
+    - imageprocessingrequestid: an autogenerated id that corresponds to the submitted job
+    - patientid: The id of the patient
+    - jobtype: The type of image processing job you want to run
+    - jobstatus: The status of the job (e.g. 'Pending', 'Running', 'Failed', 'Success')
+
+
+
+
+create an ImageProcessingRequest
+
+POST /chrisapi/v1/imageprocessingrequests
+
+Parameters
```

Comment:
do you mean something like this? 
`Type: Content
Description: An image, the id of the patient, and the type of image processing job you want to run
Required: true
Schema: Mulipart/form data: a valid medical image format (e.g. .dcm, .nii, etc.) and 2 strings
Default:`